src/Core/Checkout/Cart/Price/Struct/PriceDefinitionInterface.php

```diff
@@ -38,6 +39,8 @@ public function getPriority(): int;
      *
      * @see PercentagePriceDefinition
      * @see Rule
+     *
+     * @return array<string, Constraint[]>
```

Comment:
```suggestion
     * @return array<string, list<Constraint>>
```

list is more precise. `Constraint[]` can also be `array<string, Constraint>` or `array<int, Constraint>`